processor/tailsamplingprocessor/README.md

```diff
@@ -427,3 +427,11 @@ As a rule of thumb, if you want to add probabilistic sampling and...
 
 [probabilistic_sampling_processor]: ../probabilisticsamplerprocessor
 [loadbalancing_exporter]: ../../exporter/loadbalancingexporter
+
+## FAQ
+
+**Q. Why am I seeing high values for the error metric `sampling_trace_dropped_too_early`?**
+
+**A.** This is likely a load issue. If the collector is processing more traces in-memory than the `num_traces` configuration
+option allows, some will have to be dropped before they can be sampled. Increasing the value of `num_traces` can
+help resolve this error.
```

Comment:
```suggestion
help resolve this error, at the expense of increased memory usage.
```

Reply:
It's always a trade-off, it should be clear what are the consequences of increasing that value.

Reply:
Agreed, thanks for calling this out. 👍